src/bundler/bundle_v2.zig

```diff
@@ -11721,3 +11725,10 @@ fn targetFromHashbang(buffer: []const u8) ?options.Target {
 
     return null;
 }
+
+fn platformToUnixInPlace(path_buffer: []u8) void {
+    if (std.fs.path.sep == '/') return;
+    for (path_buffer) |*c| {
```

Comment:
please use `strings.indexOfChar` here to do this

it will use SIMD instead of probably scalar code